src/signup.css

```diff
@@ -66,4 +66,9 @@ form button[type='submit'] {
 .breadcrumb li a:hover {
   color: #01447e;
   text-decoration: underline;
+}
+
+footer {
```

Comment:
A page doesn't necessarily have one footer. Multiple sections of a website could have a footer as well. So we should use a class to style the footer.

`
<footer class="site-footer">
```

```
.site-footer {
```

Reply:
oh okay ! will update the code

Reply:
Yeah:

> The <footer> tag defines a footer for a document or section.
> You can have several <footer> elements in one document.

Ref: https://www.w3schools.com/TAgs/tag_footer.asp

In truth, we could have used just `<div>`, but we should try to use semantic tags with meaning for every section. Visually there's no difference by default.